microsoft-edge/devtools-guide-chromium/dom/index.md

```diff
@@ -185,87 +188,87 @@ To use the HTML editor:
 
    For example, try adding `<p>Dijon</p>` after the `<p>Marseille</p>` line.
```

Comment:
```suggestion
1. For example, try adding `<p>Dijon</p>` after the `<p>Marseille</p>` line.
```
tutorial type steps need to be clear whether we are or are not telling user to do an action.  if telling to do a step, needs to be a numbered paragraph.